docs/wiki/Solution creation.md

```diff
@@ -315,3 +334,105 @@ The example assumes you are using a [`bicepconfig.json`](https://docs.microsoft.
     }
 }
 ```
+
+# Pipeline-orchestration
+
+The modules provided by this repo can be orchestrated to create more complex infrastructures and as such reusable solutions or products.
```

Comment:
We should a few sentences here to explain the idea behind this orchestration option. For example:

```suggestion
The modules provided by this repo can be orchestrated to create more complex infrastructures and as such reusable solutions or products. This approach leverages the main module repository alongside its contained modules & pipeline templates to deploy resources. Each pipeline job deploys one instance of a resources and their order is controlled by specifying dependencies in the pipeline itself. 
```